extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/LightLevelSensor.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Hue Light Level Sensor -->
+	<thing-type id="0106">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Hue Light Level Sensor</label>
+		<description>A sensor providing light level information.</description>
+
+		<channels>
+			<channel id="illuminance" typeId="illuminance" />
+			<channel id="light_level" typeId="light_level" />
+			<channel id="dark" typeId="dark" />
+			<channel id="daylight" typeId="daylight" />
+			<channel id="last_updated" typeId="last_updated" />
+			<channel id="battery_level" typeId="system.battery-level" />
+			<channel id="battery_low" typeId="system.low-battery" />
+		</channels>
+
+		<representation-property>uniqueId</representation-property>
+
+		<config-description>
+			<parameter name="sensorId" type="text">
+				<label>Sensor ID</label>
+				<description>The identifier that is used within the hue bridge.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="on" type="boolean">
+				<label>Sensor Status</label>
+				<description>Indicates whether the sensor is enabled or not.</description>
```

Comment:
Does "indicate" mean that it is only for information or can we disable the sensor by changing this configuration?